hedera-node/configuration/mainnet/application.properties

```diff
@@ -1,3 +1,6 @@
+# This is exact replica of file 121 on mainnet and must match bootstrap.properties exactly.
+# It will be used by modular code for all the property overrides.
```

Comment:
@rbair23 Does "modular code" mean the "hedera-config" directory?

```suggestion
# It will be used by modular code for all the property overrides.
```

Reply:
Yes